apps/docs/package.json

```diff
@@ -3,11 +3,11 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "dev": "next dev --port 3001",
+    "dev": "next dev --port 3000",
```

Comment:
Why does someone loves changing these ports so much?

Reply:
@kilemensi This is for playwright testing in index.specs.js in pages. That we can ran a basic test to check if port works

Reply:
If it check `3000`, then it can check `3001` as well @nerdlet ... or put another way, is there any reason why it **must** be `3000`?

Reply:
@kilemensi There is no  reason at at all. We can have it at any specific port we want. The reason i did this is to ensure my playwright `page.goto()` works if i ran my playwright test. Same with web. If i ran say , both this turborepo and another project(a microservice like promisetracker) simultaneously in the background...there will be a conflict with the ports. Having to specify this info will prevent that.

Reply:
That's correct @nerdlet but we've already assign different ports to each app and hence there shouldn't be any clash. Looking at `main`:

**Web**: No port defined in [`package.json`](https://github.com/CodeForAfrica/ui/blob/main/apps/web/package.json), defaulting to `3000`. All Playwright tests in **web** should use port `3000` i.e. URLs should always start with `localhost:3000`
**Docs**: port `3001` defined in [`package.json`](https://github.com/CodeForAfrica/ui/blob/main/apps/docs/package.json). All Playwright tests in **docs** should use port `3001`.

You're _changing_ **docs** port to `3000` and hence me asking why?

Reply:
@kilemensi Alright, I removed it.

Reply:
Changes are still there @nerdlet ... compare it to `main`.